server/pkg/controller/usage.go

```diff
@@ -103,5 +104,24 @@ func (c *UsageController) CanUploadFile(ctx context.Context, userID int64, size
 			return stacktrace.Propagate(ente.ErrStorageLimitExceeded, "")
 		}
 	}
+
+	// Get particular member's storage and check if the file size is larger than the size of the storage allocated
+	// to the Member and fail if its too large.
+	var memberStorage *int64
+	if subscriptionAdminID != userID {
+		familyMembers, err := c.FamilyRepo.GetMembersWithStatus(*familyAdminID, repo.ActiveFamilyMemberStatus)
+		if err != nil {
+			stacktrace.Propagate(err, "Failed to Get Family Members")
+		}
```

Comment:
We are already fetching membersWithStatus, we should fill the value of `memberStorage` after fetching the members instead of fetching it again.